CONTRIBUTING.md

```diff
@@ -3,12 +3,15 @@
 Thank you for considering to contribute to Compiled!
 Pull requests, issues and comments are welcome.
 
-For pull requests,
-please:
+## Contribution Standards
 
 - Add tests for new features and bug fixes
 - Follow the existing style
 - Separate unrelated changes into multiple pull requests
+- As Compiled has not been officially released, when implementing a new feature, please add a changeset by calling `yarn changeset` and select `minor`.
```

Comment:
A new feature is considered a _minor_ regardless.
I suggest changing the text to
```suggestion
- As Compiled has not been officially released, when introducing a breaking change, please add a changeset by calling `yarn changeset` and select `minor`.
```